### DISCUSSION TITLE
xxxxxxxxxxxxxxxxxxx

### DISCUSSION THREAD

**Discussion by 7758521momo** [iTwin/itwinjs-core#3638] [Q&A]
[developer.bentley.com/tutorials/web-application-quick-start](https://developer.bentley.com/tutorials/web-application-quick-start/)
Follow this tutorial，I've been able to run normally，For example: Test_ imodel.

and Follow this tutorial: [https://www.itwinjs.org/learning/tutorials/create-test-imodel-offline/，I](https://www.itwinjs.org/learning/tutorials/create-test-imodel-offline/%EF%BC%8CI) have created a local snapshot.
How can I link local snapshots in “Test_ imodel”？
about file .env:

---- Authorization Client Settings ----
IMJS_AUTH_CLIENT_CLIENT_ID = ""
IMJS_AUTH_CLIENT_REDIRECT_URI = ""
IMJS_AUTH_CLIENT_LOGOUT_URI = ""
IMJS_AUTH_CLIENT_SCOPES=""

IMJS_ITWIN_ID = ""
IMJS_IMODEL_ID = ""

Thank you!

**Comment by 7758521momo** [comment]


**Reply by aruniverse** [reply]
The itwin/web-viewer-react does not support opening local snapshots. You would need to open it via the desktop viewer

**Reply by 7758521momo** [reply]
@itwin/desktop-viewer-react   
What about this？Can I open a local snapshot？
Look forward to your reply, Thank you!

**Reply by kckst8** [reply]
Yes. You can also create a new desktop application the same way that you did the web application, swapping out the web template for the desktop template `npx create-react-app my-desktop-app --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`

**Reply by kckst8** [reply]
can you be more specific about what you mean by "link local snapshots"? If you have a snapshot (.bim file), you should be able to use the File - open menu to open the file and visualize the model

**Reply by shenshengyi** [reply]
> > 您能否更具体地说明“链接本地快照”的含义？如果您有快照（.bim 文件），您应该能够使用文件 - 打开菜单打开文件并可视化模型
> 
> https://developer.bentley.com/tutorials/web-application-quick-start/ 像这样，访问在线imodel， 我想使用@itwin/desktop-viewer 或itwin/web-viewer-react 来访问本地快照， 谢谢kckst8！

你这个问题可以联系我 vx shenshengyi0926
